drivers/ioexpander/Kconfig

```diff
@@ -56,6 +56,23 @@ config IOEXPANDER_DUMMY_INT_POLLDELAY
 
 endif # IOEXPANDER_DUMMY
 
+config IOEXPANDER_ICJX
+	bool "iC-JX SPI IO expander"
+	default n
+	depends on SPI
+	---help---
+		Enable support for the iC-JX SPI IO expander.
```

Comment:
Please include comment about iC-Haus GmbH because it could help people to find the driver here

Reply:
Yes, done.